patches/net/minecraft/world/item/MobBucketItem.java.patch

```diff
@@ -14,11 +14,11 @@
 -        super(p_151138_, p_151140_);
 -        this.type = p_151137_;
 -        this.emptySound = p_151139_;
-+        this(() -> p_151137_, () -> p_151138_, () -> p_151139_, p_151140_);
++        this(() -> p_151137_, p_151138_, () -> p_151139_, p_151140_);
 +    }
 +
-+    public MobBucketItem(java.util.function.Supplier<? extends EntityType<?>> entitySupplier, java.util.function.Supplier<? extends Fluid> fluidSupplier, java.util.function.Supplier<? extends SoundEvent> soundSupplier, Item.Properties properties) {
-+        super(fluidSupplier, properties);
++    public MobBucketItem(java.util.function.Supplier<? extends EntityType<?>> entitySupplier, Fluid fluid, java.util.function.Supplier<? extends SoundEvent> soundSupplier, Item.Properties properties) {
```

Comment:
This patch can technically be entirely removed as well since we follow vanilla registration order now. In practice however there is currently a slight discrepancy between our captured vanilla registration order and the actual vanilla registration order which needs to be fixed first.

Reply:
@embeddedt remember to look into that. :smile:

Reply:
@XFactHD, now that #558 is now merged, this PR should be updated to remove this patch, correct?

Reply:
Correct, this patch can be removed now